src/elastic_derive/src/lib.rs

```diff
@@ -72,77 +74,80 @@ fn get_crate_root<'a>(item: &'a syn::MacroInput) -> Result<quote::Tokens, String
 
     match val {
         Some(crate_root) => {
-            let crate_root = get_str_from_lit(crate_root)?;
-
-            let mut tokens = quote::Tokens::new();
-            tokens.append(crate_root);
-
-            Ok(tokens)
+            // FIXME: do not hardcode the crate_root
+            Ok(quote!(crate::types).into())
```

Comment:
I can't get this to work...

I have tried:
```
let mut tokens = proc_macro2::TokenStream::new();
tokens.append_all(&[&get_string_from_lit(crate_root)?]);
Ok(tokens)
```

```
let crate_root_ident = quote::format_ident!("{}", &get_string_from_lit(crate_root)?);
Ok(quote!(#crate_root_ident).into())
```

```
Ok(get_tokens_from_lit(crate_root)?)
```

and numerous other things, but it always produces quoted strings...

Reply:
Fixed in ce53eb15f8e8f492750913dc6ab41e5b99c0c9fd